demos/Test_hydra.py

```
@@ -0,0 +1,80 @@
+import os 
+
+import stac_mjx 
+from pathlib import Path
+from jax import numpy as jp
+import mediapy as media
+import pandas as pd 
```

Ruff

`pandas` imported but unused

Remove unused import: `pandas`

(F401)